manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+    "features": {
+        "app_home": {
+          "home_tab_enabled": true,
+          "messages_tab_enabled": true,
+          "messages_tab_read_only_enabled": false
+        },
+    },
+    "oauth_config": {
+        "scopes": {
+            "bot": [
+                "channels:history",
+                "channels:read",
+                "chat:write",
+                "groups:read",
+                "im:history",
+                "im:write",
+                "mpim:read",
+                "im:read"
+            ]
+        }
+    },
+    "settings": {
+        "event_subscriptions": {
+            "bot_events": [
+                "member_joined_channel",
+                "message.channels",
+                "message.im"
+            ]
+        },
+        "interactivity": {
+            "is_enabled": true
+        },
+        "org_deploy_enabled": false,
+        "socket_mode_enabled": true,
+        "token_rotation_enabled": false
+    }
+}
```

Comment:
Slack required I include `display_information.name` and `features.bot_user.display_name`

Reply:
yes - i removed those to not have info specific to my test bot - perhaps i should just set them to. <<change me>> or something ?

Reply:
Yep, that sounds ideal to me.